packages/builder/src/pages/builder/_layout.svelte

```diff
@@ -41,13 +41,13 @@
         return
       }
 
-      if (user.tenantId !== urlTenantId) {
+      if (urlTenantId && user.tenantId !== urlTenantId) {
```

Comment:
Should be safe on the `urlTenantId` check, given the above block will return if it doesn't exist